docs/notebooks/sent2vec_tutorial.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
re-run & commit notebook please (to be 100% sure that this works with current code version).